core/src/test/java/io/grpc/internal/ServerCallImplCommonTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015, Google Inc. All rights reserved.
+ * Copyright 2017, Google Inc. All rights reserved.
```

Comment:
Yes, git automatically finds renames. In fact, git can't record a rename explicitly. Yes, sometimes it can be wrong, but in this case it seems pretty valid.